solidity/ExpirableAirdrop.sol

```diff
@@ -1,60 +1,111 @@
 // SPDX-License-Identifier: AGPL-3.0-only
-pragma solidity >=0.8.0;
+pragma solidity 0.8.7;
 
-import '@openzeppelin/contracts/access/Ownable.sol';
+import './utils/Governable.sol';
 import '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
 import '@openzeppelin/contracts/utils/cryptography/MerkleProof.sol';
 
 /// @title ExpirableAirdrop
 /// @notice airdrop claimable by members of a merkle tree with expiration date
 /// @author Mean Finance (based on: https://github.com/Anish-Agnihotri/merkle-airdrop-starter/blob/master/contracts/src/MerkleClaimERC20.sol)
-contract ExpirableAirdrop is Ownable {
+contract ExpirableAirdrop is Governable {
+  using SafeERC20 for IERC20;
+
   IERC20 public immutable token;
   uint256 public immutable expirationTimestamp;
   bytes32 public immutable merkleRoot;
   mapping(address => bool) public hasClaimed;
 
   error AlreadyClaimed();
   error Expired();
+  error NotExpired();
   error NotInMerkle();
 
-  event Claimed(address indexed to, uint256 amount);
+  event Claimed(address indexed _claimee, address indexed _receiver, uint256 _amount);
```

Comment:
```suggestion
  event Claimed(address _claimee, address _receiver, uint256 _amount);
```
I'd avoid indexing event params, since we are going to use `the graph` and emitting indexed stuff is more costly (from what I remember).